woocommerce/woocommerce-ios
# [Manual Taxes M2] New Tax Rate Selector

Closes: #10559 


## Description

With this PR we create the New Tax Rate selector screen. Buttons are working but we use demo data for the tax rates. Once we implement the fetch of the Tax Rates we will populate the screen with them.

## Testing instructions


1. Go to Orders
2. Tap on + to create a new order
3. Tap on Set New Tax Rate
4. Ensure that the navigation bar buttons and the edit tax rates in admin work fine (See video below)

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/05632b4c-b453-49a6-9a33-8d0a987b0818

---
